docs/targets/elasticsearch.md

```diff
@@ -2,47 +2,52 @@
 
 This event Target receives [CloudEvents][ce] over HTTP and writes their payload to an [Elasticsearch][es] index.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. If there is no pre-existing Elasticsearch cluster available, get started quickly by using either [Elastic Cloud on Kubernetes][eck] or [Elastic Cloud][elasticcloud].
+- An Elasticsearch cluster
 
-1. Retrieve credentials for the Elasticsearch cluster in either of the following forms:
+- Credentials for the Elasticsearch cluster in either of the following forms:
```

Comment:
Should this be just `Elasticsearch credentials` then create a section for Elasticsearch credentials like you did for `Elasticsearch cluster`

Reply:
I feel like it's too short to warrant a separate section, but I am open to hearing other opinions. I could further shorten it for the Prerequisite(s) section by wording it instead like, "- Elasticsearch credentials (username and password or API key)".